### PR TITLE
[FIX] `CHECK_CUDA` macros in debug builds

### PR DESCRIPTION
**Pull request by trxcllnt**

### CODE REVIEW

File: cpp/src/utilities/graph_utils.cuh [hunk 1]
```diff
@@ -180,7 +180,7 @@ void scatter(size_t n, T *src, T *dst, M *map)
                   thrust::device_pointer_cast(src + n),
                   thrust::device_pointer_cast(map),
                   thrust::device_pointer_cast(dst));
-  CHECK_CUDA(stream);
+  CHECK_CUDA(stream.value());
```
Comment (ChuckHastings) [inline cpp/src/utilities/graph_utils.cuh:183]: Shouldn't these next 4 be `stream_view.value()`?

**Review by seunghwak — APPROVED** [review]
We're switching from cudaStream_t to rmm::cuda_stream_view, so I think we need to eventually update CHECK_CUDA to take rmm::cuda_stream_view, but at this moment, this looks like a proper fix.

**Review by kaatish — APPROVED** [review]
LGTM

**Review by ChuckHastings — APPROVED** [review]

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**